src/components/Projects.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <v-container>
+    <DeletionDialog
+      v-model="isDeletionDialogVisible"
+      :items="[projectItem]"
+      itemsType="projects"
+    />
+    <NewProject v-model="isProjectCreationDialogVisible" />
+    <v-layout justify-center>
+      <v-flex md6>
+        <h1>Projects</h1>
+        <v-data-table
+          :headers="headers"
+          :items="availableProjects"
+          class="elevation-8"
+          :pagination.sync="pagination"
+        >
+          <template v-slot:items="props">
```

Comment:
Not my favorite variable name since props and properties have a specific separate meaning for components.

Reply:
I did not chose that as it is the default for https://vuetifyjs.com/en/components/data-tables#data-table

Reply:
I see, cheers.